src/main/java/org/reactome/release/cosmicupdate/Main.java

```diff
@@ -57,26 +64,49 @@ public static void main(String... args)
 	{
 		try
 		{
-			try(Reader configReader = new FileReader("src/main/resources/config.properties"))
+			boolean noConfigSpecified = Main.configPath != null && !Main.configPath.trim().isEmpty();
+//			boolean noAuthConfigSpecified = Main.authPath != null && !Main.authPath.trim().isEmpty();
```

Comment:
Should this (and other lines commented out below) be removed? Unless they represent future functionality

Reply:
That wasn't ready for review yet...